### PR TITLE
Improve wearable slider documentation

### PR DESCRIPTION
**Pull request by lslachciak**
[Problem] Wearable slider documentation did not provide useful information
[Solution] Edit slider widget documentation

Signed-off-by: Pawel Kaczmarczyk <p.kaczmarczy@samsung.com>

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/592

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm [hunk 1]
```diff
@@ -1,49 +1,451 @@
...
+
+					</div>
+
+						<div class="example">
+							<span class="example"><p>Code
+								example :</p><p></p></span>
+							<pre name="code" class="examplecode
+							prettyprint">
+&lt;input id=&quot;Slider&quot; name=&quot;slider-1&quot; type=&quot;range&quot; value=&quot;5&quot; min=&quot;0&quot; max=&quot;10&quot;/&gt;
+&lt;script&gt;
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: the same as above

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm [hunk 2]
```diff
@@ -1,49 +1,451 @@
...
+	</table>
+
+
+
+<h2><a id="methods0.2250751326791942"></a>Methods</h2>
+
+<p>To call a method on the component, use one of the existing APIs:</p>
+
+<pre class="prettyprint">&lt;input id=&quot;slider&quot; type=&quot;range&quot; name=&quot;slider-1&quot; value=&quot;60&quot; min=&quot;0&quot; max=&quot;100&quot;/&gt;
+&lt;script&gt;
```
[the remark this answers is not in the document]
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: Done

**Comment by lslachciak** [comment]
@dorlinski could you rebase your last commit?

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm [hunk 3]
```diff
@@ -1,49 +1,475 @@
...
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">buttons</span></td>
+				<td class="option">boolean</td>
+				<td class="option">false</td>
+				<td class="option">Enable additional + / - buttons</td>
+			</tr>
+
+			<tr>
+				<td class="option"><span style="font-family: Courier New,Courier,monospace">containerClassName</span></td>
+				<td class="option">?string</td>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: Is it "?string" or "string"?

**Comment by mijin85cho** [comment]
Please rebase to resolve conflict as @TomaszLukawskiSam asked.

**Comment by dorlinski** [comment]
Done

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm [hunk 4]
```diff
@@ -1,58 +1,475 @@
...
+
+		</li>
+
+		<li><a href="#methods0.2250751326791942">Methods</a>
+
+		</li>
+
+</ol>
+
+<h2><a id="default-selectors0.42985791829414666"></a>Default Selectors</h2>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: This header "Default Selectors" is not included in the toc list of line 18. Did you miss it by intent?

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm [hunk 5]
```diff
@@ -1,58 +1,475 @@
...
+
+<h2><a id="manual-constructor0.6728984923101962"></a>Manual Constructor</h2>
+
+<p>To manually create a slider component, use the component constructor from the <code>tau</code> namespace</p>
 
-<p>To add a slider component to the application, use the following code:</p>
+<p>HTML code:</p>
 
-<video autoplay muted loop controls>
-	<source src="https://developer.tizen.org/sites/default/files/documentation/slider.mp4">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: This mp4 file is added only few days ago. Are you going to remove this video now?

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: I think it should stay

**Comment by dorlinski** [comment]
I squashed two commits together and repiared it.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm [hunk 6]
```diff
@@ -1,58 +1,480 @@
...
-     <script type="text/javascript" src="../snippet.js"></script>
-     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-     <link href="../styles.css" rel="StyleSheet" type="text/css" />
-     <link href="../snippet.css" rel="StyleSheet" type="text/css" />
-     <title>Slider</title>
- </head>
- <body id="content">
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: Question. Was there any reason for changing "/>" to ">"?

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: @pkaczmarczy

Comment (pkaczmarczy) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: This documentation page has been built based on mobile slider documentation and header has been copied.
However <meta> and <link> are void elements, according to https://www.w3.org/TR/html5/syntax.html#start-tags item 6 the "/"  is optional and it has no effect.

Comment (mijin85cho) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: Thanks for the description.
@TomaszLukawskiSam , please check the latest version and approve it if this is no issue in this PR.

Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm]: I accept @pkaczmarczy 's explanation. I accept PR.

**Pull request merged**